src/modules/blog/Blog.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Request, Response } from "express";
+import { BlogDto } from "./dto/blog-post.dto";
+import { Blog } from "./Blog.model";
+import { MongoClient } from "mongodb";
+import mongoose from "mongoose";
+
+
+export class BlogService {
```

Comment:
you have two options here. 
1.  return res.status(201).json({
            message: "Blog successfully created",
            blog: newBlog
        })
        
        This can be moved to controller. 
2. If you want to leave it here, implement error handling. what if there is error in any of the operations. user must be properly informed. You handle only success cases.